static/dependencies.config.php

```diff
@@ -224,6 +224,9 @@
 			['start', [], Dice::CHAIN_CALL],
 		],
 	],
+	\Friendica\Core\Session\Capability\IHandleUserSessions::class => [
```

Comment:
Hmm, I would choose to move this to the header of the file and "import" it with `use` instead.